### PR TITLE
Delete UITips from teacher homepage and courses

### PR DESCRIPTION
**Pull request by Erin007**
About 2 years ago when we redesigned the homepage and courses experience, Brendan added in [tooltips explaining the changes to users](https://github.com/code-dot-org/code-dot-org/pull/15463). When a user first arrived on their homepage, they saw a dialog and tool tips pointing out new features. 
Screen Shot 2019-05-01 at 10 50 24 AM

Shortly after launching these tooltips, it was decided that[ new users didn't need to see these tooltips](https://github.com/code-dot-org/code-dot-org/pull/15957). 

This means the only people who would currently still see this feature are users who were active prior to the redesign but haven't been active since; therefore the tooltips are basically dead code, so I deleted them and all of the references to them.

### CODE REVIEW

**Comment by breville** [comment]
I wonder if we should clean out the flag from the User model in the database as well?

**Comment by Erin007** [comment]
> I wonder if we should clean out the flag from the User model in the database as well?

We should, but I'd like to do it as a follow up step.  We'd need to run a migration on the users table, which I think takes a good chunk of time and has the potential to be a little scary.

**Comment by breville** [comment]
@Erin007 I don't think we'll need to do a migration since this is a serialized attribute, but you're right that it can be done as a followup.

**Comment by breville** [comment]
I still remember the long weekend where I built this!  So much work for so few tips :)  

Thanks for noticing and taking the time to clean it up.

**Review by breville — APPROVED** [review]

File: dashboard/app/views/home/index.html.haml [hunk 1]
```diff
@@ -4,22 +4,18 @@
...
-
 -# Student modal
 - if (current_user && current_user.show_race_interstitial?) || @force_race_interstitial
   = render partial: 'layouts/race_interstitial'
 
 -# Teacher modals
 - if current_user && current_user.teacher? && !current_user.accepted_latest_terms?
   = render partial: 'layouts/terms_interstitial'
-- elsif (current_user && current_user.show_school_info_interstitial?) || @force_school_info_interstitial
+- if (current_user && current_user.show_school_info_interstitial?) || @force_school_info_interstitial
```
Comment (breville) [inline dashboard/app/views/home/index.html.haml]: I think this should remain `elsif` so that we don't show more than one interstitial.

**Pull request merged**